packages/artifacts/chunks/languages/builtins-types.ts.txt

```diff
@@ -28,3 +25,4 @@ export type JSON = unknown;
 
 export type String = string;
 
+export type LionCore-builtinsNode = ;
```

Comment:
Is this valid TS? No assigned value?

Reply:
Not correct, also because of the invalid ID. Fixed this just now.